dist/index.html

```diff
@@ -40,29 +40,31 @@ <h1> What's cookin? </h1>
       <section class="recipe-details" id="recipeLeftSide">
         <div class="modal-btn-container">
           <button class= "add-recipe modal-recipe-btn">
-            <img class="bookmark-img" src="./images/hollow-bookmark-icon.png" id="add">
+            <img tabindex="0" class="bookmark-img" src="./images/hollow-bookmark-icon.png" id="add">
           </button>
           <button class="remove-recipe modal-recipe-btn hidden" id="remove">
-            <img class="bookmark-img" src="./images/select-bookmark-icon.png" id="remove">
+            <img tabindex="0" class="bookmark-img" src="./images/select-bookmark-icon.png" id="remove">
           </button>
           <p class="modal-feedback"></p>
-          <img class="close-btn" id="closeRecipeSmall" src="delete.png"></img>
+          <img tabindex="1" role="button" class="close-btn" id="closeRecipeSmall" src="delete.png"></img>
         </div>
         <div id="recipeName"></div>
         <article id="recipeInstructions">
           <!-- recipe instructions will appear here -->
         </article>
       </section>
       <section class="recipe-details" id="recipeRightSide">
-        <img  class="close-btn" id="closeRecipe" src="delete.png"></img>
+        <img  tabindex="0" role="button" class="close-btn" id="closeRecipe" src="delete.png"></img>
         <h3>Ingredients</h3>
-        <div class="ingredients-list">
+        <article tab-index="0" class="ingredients-list">
           <!-- recipe ingredients will appear here  -->
-        </div>
+        </artcile>
       </section>
     </section>
     <section id="graphPanel" class="info-panel hidden">
-      <img id="closeGraph" class="close-btn" src="delete.png"></img>
+      <button id="closeGraphBtn"> 
+        <img tabindex="0" id="closeGraph" class="close-btn" src="delete.png"></img>
```

Comment:
I notice that everywhere else, images have the role of a button.
Here, it's wrapped in a button div. Is there a reason for this?

Reply:
I needed to focus this element, and buttons are focusable but images are not. 

We should have made all of the images this way (wrapped in a button), but since we didn't the styling and JS functionality would be a lot to change at this point, so giving them roles of button will suffice. I couldn't get around that here because I needed the button to get focused when the graph modal opens in order to allow tabbing inside the modal.